tests/devices_test/device_packages_test.py

```diff
@@ -18,9 +18,12 @@ def testPackages(self):
         dev = MDRaidArrayDevice("dev", level="raid1", parents=[dev1,dev2])
         luks = LUKSDevice("luks", parents=[dev])
         packages = luks.packages
+        packages.sort() # sort due to blivet/issues/154
 
         # no duplicates in list of packages
-        self.assertListEqual(packages, list(set(packages)))
+        l = list(set(packages))
+        l.sort() # sort due to blivet/issues/154
+        self.assertListEqual(packages, l)
```

Comment:
> We can convert both to sets and compare the sets.

I fail to see how this can be done. 
```
>>> set([1, 2, 3, 2, 1]) == set([1, 2, 3])
True
```

How do you figure out if the first set had duplicate elements in in, please explain ?